subsys/net/lib/sockets/sockets.c

```diff
@@ -49,7 +49,7 @@ static void zsock_received_cb(struct net_context *ctx,
 			      int status,
 			      void *user_data);
 
-static inline int _k_fifo_wait_non_empty(struct k_fifo *fifo, int32_t timeout)
+static inline int k_fifo_wait_non_empty(struct k_fifo *fifo, int32_t timeout)
```

Comment:
The naming of this function is not that great atm. This private function looks now like a global function. Imho we need to change the name to something like `wait_non_empty()` or `fifo_wait_non_empty()`. This could be done in this PR or I can send a separate PR after this one is merged.